### PR TITLE
Inline install firefox extension

### PR DESCRIPTION
**Pull request by danjenkins**
This PR adds in the built firefox extension and a way of installing it from firefox without having to put the extension into the mozilla extension store

### CODE REVIEW

File: app/modules/screen-sharing/controllers/AuthenticationCtrl.js [hunk 1]
```diff
@@ -28,15 +28,34 @@ App.controllers.authenticationCtrl = (function ($, App) {
...
+                    console.error('Chrome webstore URL is', chromeUrl);
+                });
+            }
+
+            if(respoke.needsFirefoxExtension && !respoke.hasFirefoxExtension) {
+                console.log('attempting to install firefox extension');
+                InstallTrigger.install({
+                    Foo: {
+                        URL: '/web-examples-respoke.xpi',
+                        //IconURL: aEvent.target.getAttribute("iconURL"),
```
Comment (chadxz) [inline app/modules/screen-sharing/controllers/AuthenticationCtrl.js]: do we not have an icon for it?

**Comment by chadxz** [comment]
looks good :+1:

**Comment by chadxz** [comment]
does it work? :)

**Comment by pho3nixf1re** [comment]
Does it blend?

**Pull request merged**